docs/TektonResult.md

```diff
@@ -148,3 +148,22 @@ stringData:
   S3_MULTI_PART_SIZE: "5242880"
 ```
 
+
+### GCS specific Property
```

Comment:
where to add these, please mention in the results spec

Reply:
I think it's clear now. Please check.